src/integTest/java/ai/deepcode/javaclient/DeepCodeRestApiTest.java

```diff
@@ -75,12 +77,9 @@ public void _020_checkSession() {
     System.out.printf(
         "Check Session call with newly requested but not yet logged token [%1$s] return [%2$d] code.\n",
         token, status);
```

Comment:
That should be removed as we removed actual test for such case

Reply:
cood catch